pkg/gce-pd-csi-driver/node.go

```diff
@@ -320,25 +317,25 @@ func (ns *GCENodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpa
 }
 
 func (ns *GCENodeServer) GetVolumeLimits() (int64, error) {
-	// Default value 16 for shared-CPU VM
+	// Default value 16 for memory lower than cutoff
 	volumeLimits := VolumeLimit16
 
-	// Machine-type format: n1-type-NCPU
+	// Machine-type format: n1-type-CPUS or custom-CPUS-RAM
 	machineType := ns.MetadataService.GetMachineType()
 	if strings.HasPrefix(machineType, "n1-") {
+		volumeLimits = VolumeLimit128
+	} else if machineType == "f1-micro" || machineType == "g1-small" {
```

Comment:
Can you add unit tests for all of these?

Reply:
OK, I will have a try

Reply:
For the unit test, I add a function to set machine type for fake metaserver. I'm not sure if it's an appropriate way to do so.